### PR TITLE
IN BODY PR CHANGELOGS!

### PR DESCRIPTION
**Pull request by PJB3005**
🆑 
- rscadd: Adds in PR body changelogs. No more messing around with dumb YAML files and such when making changelogs. This is mostly for us coders by the way.

### CODE REVIEW

**Comment by Exxion** [comment]
Note that it's already implemented. This PR just adds something to the template to tell us how to use it and, if it works, adds a changelog entry.

**Comment by Exxion** [comment]
It didn't.

**Comment by PJB3005** [comment]
Yeeep I fucked up, found where it fucked up and fixed it though.

The bug in the changelog script's been around for actual ages in fact but this just made it 10x worse so I finally realized.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -10,5 +10,12 @@ Large changes require discussion.  If you're doing a large, game-changing modifi
 Merging your own PRs is considered bad practice, as it generally means you bypass peer review, which is a core part of how we develop.
```
Comment (ihadtoregisterforthis) [inline .github/PULL_REQUEST_TEMPLATE.md:10]: :joy: :ok_hand:

Comment (Exxion) [reply .github/PULL_REQUEST_TEMPLATE.md:10]: You're saying this like it's not common knowledge

**Pull request merged**